Client/Frontend/Fakespot/Views/HighlightsCardView.swift

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import ComponentLibrary
+import Common
+import UIKit
+
+struct HighlightsCardViewModel {
```

Comment:
You added an init for `ActionFooterViewModel`, I suppose for consistency you have to add another one here.